servers/eap7.2/src/main/java/org/jboss/migration/eap/task/hostexclude/EAP7_2AddHostExcludes.java

```diff
@@ -25,81 +25,229 @@
+                    <extension module="org.wildfly.extension.core-management"/>
+                    <extension module="org.wildfly.extension.datasources-agroal"/>
+                    <extension module="org.wildfly.extension.discovery"/>
+                    <extension module="org.wildfly.extension.ee-security"/>
+                    <extension module="org.wildfly.extension.elytron"/>
+                    <extension module="org.wildfly.extension.microprofile.config-smallrye"/>
+                    <extension module="org.wildfly.extension.microprofile.health-smallrye"/>
+                    <extension module="org.wildfly.extension.microprofile.opentracing-smallrye"/>
+                </excluded-extensions>
+            </host-exclude>
+             */
             .hostExclude(HostExclude.builder()
                     .name("EAP70")
                     .release("EAP7.0")
                     .excludedExtension("org.wildfly.extension.core-management")
+                    .excludedExtension("org.wildfly.extension.datasources-agroal")
                     .excludedExtension("org.wildfly.extension.discovery")
                     .excludedExtension("org.wildfly.extension.ee-security")
-                    .excludedExtension("org.wildfly.extension.elytron"))
+                    .excludedExtension("org.wildfly.extension.elytron")
+                    .excludedExtension("org.wildfly.extension.microprofile.config-smallrye")
+                    .excludedExtension("org.wildfly.extension.microprofile.health-smallrye")
+                    .excludedExtension("org.wildfly.extension.microprofile.opentracing-smallrye"))
+            /*
+            <host-exclude name="EAP71">
+                <host-release id="EAP7.1"/>
+                <excluded-extensions>
+                    <extension module="org.wildfly.extension.datasources-agroal"/>
+                    <extension module="org.wildfly.extension.ee-security"/>
+                    <extension module="org.wildfly.extension.microprofile.config-smallrye"/>
+                    <extension module="org.wildfly.extension.microprofile.health-smallrye"/>
+                    <extension module="org.wildfly.extension.microprofile.opentracing-smallrye"/>
+                </excluded-extensions>
+            </host-exclude>
+             */
             .hostExclude(HostExclude.builder()
                     .name("EAP71")
                     .release("EAP7.1")
-                    .excludedExtension("org.wildfly.extension.ee-security"))
+                    .excludedExtension("org.wildfly.extension.datasources-agroal")
+                    .excludedExtension("org.wildfly.extension.ee-security")
+                    .excludedExtension("org.wildfly.extension.microprofile.config-smallrye")
+                    .excludedExtension("org.wildfly.extension.microprofile.health-smallrye")
+                    .excludedExtension("org.wildfly.extension.microprofile.opentracing-smallrye"))
```

Comment:
I've just seen this PR; it looks like it is still missing `org.wildfly.extension.microprofile.metrics-smallrye` in those exclusions